### DISCUSSION TITLE
How to install config?

### DISCUSSION THREAD

**Discussion by kurisutinade** [zDyant/HyprNova#10] [Q&A]
Topic

**Comment by zDyant** [comment]
Hey! Each part of this repo contains a config for each program, either you may use a certain config, or you can use the entire config copying the entire folder `~/.config` to your home

remember to run pywal using `wal <wallpaper>` afterwards

**Reply by kurisutinade** [reply]
I moved all the files from /HyprNova/.config/ to /home/user/.config/ and started hyprland and it gives errors like this
![Capture](https://github.com/user-attachments/assets/724a5232-b74c-4ba2-8262-86c30bf1f142)

**Reply by zDyant** [reply]
it's missing pywal file, use `wal` to generate it,
remove `new_is_master` and `no_direct--` from the config,
that should do it

**Reply by azizsayfiddin** [reply]
> Hey! Each part of this repo contains a config for each program, either you may use a certain config, or you can use the entire config copying the entire folder `~/.config` to your home
> 
> remember to run pywal using `wal <wallpaper>` afterwards

Which wallpapers to choose?

**Reply by zDyant** [reply]
You can choose, however I have a repository with wallpapers that I use
